kahuna/public/stylesheets/main.css

```diff
@@ -779,6 +779,10 @@ input.search-query__input {
     z-index: 1;
 }
 
+.result__select--allow-shift-click {
```

Comment:
Class name is a little confusing to me - it seems too strongly coupled to the controller logic? Could make it more generic so it can be used elsewhere? e.g. `no-pointer-events`.